src/index.html

```diff
@@ -5,10 +5,43 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;0,700;1,500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="wrap-logo">
+        <img
+          src="./images/logo.png"
+          alt="Mlogo"
+          class="fixed-size-image"
+        >
+      </div>
+      <nav class="nav-container">
+        <ul class="nav-list">
+          <li><a class="nav-link1 is-active" href="#apple">Apple</a></li>
```

Comment:
Need to fix code style everywhere and add the classes for the tag li
```suggestion
          <li class="nav__item">
            <a class="nav-link is-active" href="#apple">Apple</a> 
          </li>
```